staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -112,6 +113,9 @@ func (s *ServerRunOptions) DefaultAdvertiseAddress(secure *SecureServingOptions)
 func (s *ServerRunOptions) Validate() []error {
 	errors := []error{}
 
+	if s.AdvertisePort < 0 || s.AdvertisePort > 65535 {
+		errors = append(errors, fmt.Errorf("--advertise-port %v must be between 0 and 65535, inclusive. 0 for turning off", s.AdvertisePort))
```

Comment:
What does "turning off" mean for AdvertisePort?

Reply:
Opted to remove this trailing bit here, as the actual flag's description already states what the default behaviour is if not set (`If not specified the --secure-port or --port is used` below), and I don't think it makes so much sense to include it in the `Validate` text that only displays if a port number is out of range.